README.md

```diff
@@ -6,6 +6,17 @@ invite and meet link.
 
 Send to: llvm@lists.linux.dev
 
+## Mar 8, 2023
+- (Nick) scaling down CI
+- (Nick/Bill) ["p" constraint](https://lore.kernel.org/lkml/20230303231133.1486085-1-eranian@google.com/)
+- (Connor) [riscv breakages due to extensions moving out of I spec](https://github.com/ClangBuiltLinux/linux/issues/1808)
+- [riscv 5.10 build is broken when using BFD
```

Comment:
These are the same issue

Reply:
oh, I thought I had deleted this line when I realized that. I must hit ctrl+z at some point!